bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Shell.java

```diff
@@ -2650,19 +2645,8 @@ LRESULT WM_WINDOWPOSCHANGING (long wParam, long lParam) {
 	return result;
 }
 
-/**
- * The native zoom in % of the standard resolution the shell is scaled for
- */
-int getNativeZoom() {
-	return nativeZoom;
-}
-
-void setNativeZoom(int nativeZoom) {
-	this.nativeZoom = nativeZoom;
-}
-
 private void handleZoomEvent(Event event) {
-	float scalingFactor = 1f * event.detail / getZoom();
+	float scalingFactor = 1f * DPIUtil.getZoomForAutoscaleProperty(event.detail) / getZoom();
```

Comment:
Could you please assign the `event.detail` value to a variable with a self-explaining name to make this more comprehensive?